bower.json

```diff
@@ -36,8 +38,7 @@
       },
       "devDependencies": {
         "iron-demo-helpers": "PolymerElements/iron-demo-helpers#^1.0.0",
-        "promise-polyfill": "polymerlabs/promise-polyfill#1 - 2",
-        "test-fixture": "PolymerElements/test-fixture#^1.0.0",
+        "promise-polyfill": "PolymerLabs/promise-polyfill#^1.0.0",
```

Comment:
This should actually be `1 - 2` since the 2.0 just decouples the polymer dep. I believe I checked this with @e111077 back when I was making the changes.

Reply:
Then why not `^2.0.0`?

Reply:
we could, just figured it didn't really matter since there isn't incompatibility in the actual promise polyfill. I'm open to either